packages/cli/create-strapi-starter/utils/build-starter.js

```diff
@@ -29,11 +28,12 @@ function readStarterJson(filePath, starter) {
 }
 
 /**
- * @param  {string} rootPath - Path to the project directory
- * @param  {string} projectName - Name of the project
- * @param  {boolean} useYarn - Use yarn instead of npm
+ * @param {Object} options
+ * @param {string} options.rootPath - Path to the project directory
+ * @param {string} options.projectName - Name of the project
+ * @param {boolean} options.useYarn - Use yarn instead of npm
  */
-async function initPackageJson(rootPath, projectName, useYarn) {
+async function initPackageJson({ rootPath, projectName, useYarn }) {
```

Comment:
Sorry about the misunderstanding I meant `initPackageJson(path, name, { usYarn })` only passing an options object as the 3 parameter as this one could evolve but not the first two :)